amazonredshift_blueprints/upload_file.py

```diff
@@ -251,14 +216,12 @@ def main():
         for index, key_name in enumerate(matching_file_names):
             upload_data(
                 source_full_path=key_name,
-                table_name=table_name,
```

Comment:
Why did you remove the `table_name` parameter?

Reply:
good catch- i believe this was a selection error when removing the database param